File: src/Services/User/User.Domain/Entities/LoanLead.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using User.Domain.Common;

namespace User.Domain.Entities
{
    public class LoanLead:EntityBase
    {
        public DateTime LoanDate {  get; set; }
        public string LoanBorrower { get; set; }
        public string AdharNumber { get; set; }
        public string OrganizationCode { get; set; }
        public string LoanType { get; set; }
        public string? LeadStage { get; set; }
        public string? LeadStatus { get; set; }
        public string? LeadStatusRemarks { get; set; }
        public string? VoterId { get; set; }
        public string? PanCardNumber { get; set; }
        public int Amount { get; set; }
    }
}

File: src/WebApps/clr-portal/src/app/loans/model/loanlead.ts
export interface LoanLead {
  amount: string;
  organizationCode: string;
  organizationName: string;
  adharNumber: string;
  loanDate: Date;
  loanBorrower: string;
  loanType: string;
  id: number;
  leadStage: string;
  leadStatus: string;
  leadStatusRemarks: string;
}

export function compareLoans(c1: LoanLead, c2: LoanLead) {
  const compare = c1.id - c2.id;

  if (compare > 0) {
    return 1;
  } else if (compare < 0) {
    return -1;
  } else return 0;
}
